projects/gen-ai.md

```diff
@@ -0,0 +1,143 @@
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+- develop common instrumentation libraries for popular GenAI clients and frameworks
+- suggest and, when possible, contribute them to upstream libraries
+- define and evolve GenAI Semantic Conventions
+- propose and implement changes in the OTel API/SDK/contrib/etc that are necessary for GenAI efforts
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel LLM SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+- Python
+  - maintainers/approvers:
+    - @drewby
+    - @karthikscale3
+    - @lzchen
+    - @nirga
+  - contributors:
+    - @alizenhom
+    - @lmolkova
+    - *looking for more contributors*
+- JS
+  - maintainers/approvers:
+    - @drewby
+    - @karthikscale3
+    - @nirga
+    - *looking for existing OTel JS approvers/maintainers for general JS & OTel feedback*
+  - contributors:
+    - @alizenhom
+    - *looking for more contributors*
+- Other languages: not in the scope (yet).
+
+## Meeting Times
+
+Alternating weekly meetings to accommodate different time zones:
+
+- Week 1: Wednesdays at 10 AM PST
+- Week 2: Wednesdays at 11 PM PST
+
+(will likely add more slots depending on the demand)
+
+### Meeting Links
+
+- Meeting Calendar: [calendar-semconv@opentelemetry.io](https://groups.google.com/a/opentelemetry.io/g/calendar-semconv)
```

Comment:
People that join this group will get all invites to every single semconv event. Would you prefer to create a different group now that it's more than semconv?

Reply:
Based on the past discussions in the SIG, there are a lot of discussions around instrumentations. Semconv discussions are hard to separate from questions around what we need and can capture, how to do it, what's available, etc. 

I'll check with the LLM semconv group but I'd prefer to keep it all as one project and evolve semantic conventions along with instrumentations.